app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>List of Events you are attending</h1>
+
+<h2>Upcoming events</h2>
+<ul>
+  <% current_user.attended_events.where('date >= ?', DateTime.now).each do |event| %>
+    <li>
+      <%= event.name%>
+    </li>
+  <% end %>
+</ul>
+
+<h2>Past events</h2>
+<ul>
+  <% current_user.attended_events.where('date < ?', DateTime.now).each do |event| %>
```

Comment:
Kindly use the scope from Event model and pass it as an instance variable to your views.
`@past_events = current_user.attended_events.previous`

Reply:
Thanks. I have implemented this change similar to the one above by using the Event scope past method.